package.json

```diff
@@ -8,7 +8,7 @@
     "build": "tsc",
     "format": "prettier --write **/*.ts",
     "format-check": "prettier --check **/*.ts",
-    "release": "ncc build && git add -f dist/",
+    "release": "ncc build lib/setup-java.js && git add -f dist/index.js",
```

Comment:
Why have `ncc` build the javascript files? It can handle typescript

Reply:
Nice catch! The `.ts` source file can be used. See the following example: https://github.com/zeit/ncc/blob/bfefbbce9f5bdf5ba9514fd1eb0c133aa3ba3d32/examples/typescript/package.json#L8

Reply:
Updated this.  Please double check as I'm new to `ncc`

Reply:
What you have is works and I tested it out. There are quite a few ways to configure ncc in `package.json`. Relatively new to `ncc` as well 😁

 Might change it up a little bit to align it with some of our other actions that also use `ncc` like `actions/cache` but I'll handle that myself in another PR. Thanks!